ksqldb-engine/src/test/java/io/confluent/ksql/query/QueryRegistryImplTest.java

```diff
@@ -24,6 +24,7 @@
 import io.confluent.ksql.query.QueryRegistryImpl.QueryExecutorFactory;
 import io.confluent.ksql.schema.ksql.LogicalSchema;
 import io.confluent.ksql.services.ServiceContext;
+import io.confluent.ksql.util.KsqlConstants;
```

Comment:
Can we test the query gets removed from only the correct list here? It seems that would cover the case that was causing the the problem

Reply:
Done